alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/action/SearchMonitorAction.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.alerting.action
+
+import org.elasticsearch.action.ActionType
+import org.elasticsearch.action.search.SearchResponse
+
+class SearchMonitorAction private constructor() : ActionType<SearchResponse>(NAME, ::SearchResponse) {
```

Comment:
Where is the test for this class? We only have a test for `DeleteMonitorAction`.

Reply:
Added